recipes/spatialist/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "spatialist" %}
+{% set version = "0.5" %}
+{% set maintainer = "johntruckenbrodt" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89c56e987b79b5b904eb5d22482f3addf0cd3c60bda9910680ebc478b10520ee
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools_scm
+  run:
+    - python
+    - progressbar2
+    - jupyter
+    - matplotlib
+    - pathos >=0.2
+    - numpy
+    - tblib
+    - pyyaml
+    - gdal >=2.1
+    - sqlite
+    - libspatialite
+    - setuptools_scm
+
+test:
+  source_files:
+    - spatialist/tests
+  requires:
+    - pytest
+  imports:
+    - {{ name }}
```

Comment:
For the `imports`  down to the end of the `meta.yaml`, it preferable to not use jinja templating unnecessarily. It just makes it harder to look at the recipe and then figure out links before the recipe is rendered

Reply:
Sure. Done. I'll remember that for future packages.